Makefile

```diff
@@ -90,9 +92,7 @@ MODIFIED_TYPESCRIPT_FILES = $(filter %.ts %.json %.yml,$(MODIFIED_FILES))
 
 format: install_rustfmt install_tomlfmt
 	$(CARGO_NIGHTLY) fmt -- --files-with-diff | xargs -I{} git add {}
-	RUST_LOG=error $(CARGO) tomlfmt -p Cargo.toml && git add Cargo.toml
-	RUST_LOG=error $(CARGO) tomlfmt -p cnd/Cargo.toml && git add cnd/Cargo.toml
-	RUST_LOG=error $(CARGO) tomlfmt -p libp2p-comit/Cargo.toml && git add libp2p-comit/Cargo.toml
+	@$(foreach file,$(CARGO_TOML_FILES),$(CARGO) tomlfmt -p $(file) && git add $(file);)
```

Comment:
:mega: I think you dropped the `RUST_LOG` now right? Can you add that back please?

Reply:
Can you explain the utility of the `RUST_LOG` here?

Reply:
To silence the output of tomlfmt.

Reply:
Have you tested locally? I achieve the wanted result with the `@` at the start of the line.

Reply:
I did not!

I trust you that you figured out this Makefile voodoo :)